src/main/java/taxi/service/DriverService.java

```diff
@@ -1,6 +1,8 @@
 package taxi.service;
 
+import taxi.exception.AuthenticationException;
 import taxi.model.Driver;
 
 public interface DriverService extends GenericService<Driver> {
+    Driver findByLogin(String login) throws AuthenticationException;
```

Comment:
Change the method signature to remove 'throws AuthenticationException' as it is not necessary for the 'findByLogin' method in the interface